smartsim/database/cobaltOrchestrator.py

```diff
@@ -149,6 +150,21 @@ def set_batch_arg(self, arg, value):
             raise SmartSimError("Not running as batch, cannot set batch_arg")
         self.batch_settings.batch_args[arg] = value
 
+    def set_run_arg(self, arg, value):
+        """Set a run argument the orchestrator should launch
+        each node with (it will be passed to `aprun`)
+        
+        Some commonly used arguments are used 
+        by SmartSim and will not be allowed to be set.
```

Comment:
Should we restrict the user from changing ``pes``? as in the number of tasks for the ``Orchestrator``? 

I feel like there might be a few settings that would break the ``Orchestrator`` launch.

Reply:
``--wdir`` is one.